controllers/openstackmachine_controller.go

```diff
@@ -446,7 +489,38 @@ func (r *OpenStackMachineReconciler) reconcileNormal(ctx context.Context, scope
 	return ctrl.Result{}, nil
 }
 
-func (r *OpenStackMachineReconciler) getOrCreate(logger logr.Logger, cluster *clusterv1.Cluster, openStackCluster *infrav1.OpenStackCluster, machine *clusterv1.Machine, openStackMachine *infrav1.OpenStackMachine, computeService *compute.Service, userData string) (*compute.InstanceStatus, error) {
+func getOrCreateMachinePorts(scope scope.Scope, openStackCluster *infrav1.OpenStackCluster, machine *clusterv1.Machine, openStackMachine *infrav1.OpenStackMachine, networkingService *networking.Service, clusterName string) error {
+	scope.Logger().Info("Reconciling ports for machine", "machine", machine.Name)
```

Comment:
Is this important enough to be info level? I don't have a strong opinion, just a thought

Reply:
I think it should be enough with changes, meaning if a port is created/deleted or updated?

Reply:
`Info` is already used elsewhere in the controller, for example when handling machines or also load balancers, etc. I think this is fine for now.

Reply:
Yeah I know it is used for "Reconciling machine" for example. It just feels to me like "ports for machine" would be already implied by "Reconciling machine", as in this is a lower level. Load balancers on the other hand is not part of that machine so it makes sense that they get a separate log message.

If we end up creating or changing a port during this reconciliation I would definitely expect an `Info` level message for that, but this is just information that we made it this far in the loop.

Reply:
Good catch. I'll fix that right on a next iteration.